int32 disparity
float32 a
float32 b
---
float32 depth
